src/codelabs/async-await/index.md

```diff
@@ -189,13 +184,24 @@ First, add the `async` keyword before the function body.
 {% endprettify %}
 
 If needed, update the function's type signature to return a future.
+If the function has a declared return type, then update the type to be `Future<T>`,
+where `T` is the type of the value that the function returns. 
+If the function doesn't explicitly return a value, then the return type is `Future<void>`:
+
 {% prettify dart %}
     [!Future<void>!] main() async {
```

Comment:
It's a little confusing to me that you added Future<void> before the main() but didn't include this in the later side-by-side example. 

The content says "If needed, update the function’s type signature". This makes me wonder "so why did you add a function’s type signature here but didn't add a function’s type signature in "Example: asynchronous functions".

Reply:
I'm removing the `If needed, update the function's type signature" sentence.

Reply:
I agree with your point about `you added Future before the main() but didn't include this in the later side-by-side example.`, which is why I think `main()` might not be the best choice for this example. @kathyw thoughts? It seems sort of odd (but not wrong) to show examples with `Future<void> main()` everywhere, but it also seems odd to do it in some places but not others.

Reply:
I'm curious about the rationale of changing from `converts createOrderMessage ()` to `concerts main()`.

Reply:
I'm trying to remember why we changed from `createOrderMessage()` to `main()`. I agree that it might be better to show something that isn't `main()`, so we don't have to dance around the return type declaration being OK to leave off. (Or we can do the dance later.)

I think the text was inconsistent about `Future<void>` vs. `Future<TypeOfWhatYouReturn>`, so we picked `Future<void>`. Also, maybe we wanted to handle the no-return case? But we could do that separately.

Reply:
> Also, maybe we wanted to handle the no-return case? But we could do that separately.

It seems that both examples in the [What is a future?](https://jt-ww-1.firebaseapp.com/codelabs/async-await#what-is-a-future) section has handled the no-return case using Future<void>.